doc/Language/faq.pod6

```diff
@@ -370,6 +370,99 @@ L<.comb|/routine/comb> is accepting an optional L<Int|/type/Int>:
 .say for 'abcdefghijklmnopqrstuvwxyz'.comb: 8;  # OUTPUT: «abcdefgh␤ijklmnop␤qrstuvwx␤yz»
 =end code
 
+=head2 List: How can I remove duplicates items in a list
+
+Use L<.unique|/routine/unique>:
+
+=begin code
+(1, 2, 3, 2, 3, 1, 1, 0).unique;  # OUTPUT: «(1 2 3 0)␤»
+=end code
+
+=head2 List: How can I delete an element from a list
+
+To delete by index, use L<.splice|/routine/splice>:
+
+=begin code
+my @a = <3 18 4 8 92 14 30>;
+say @a.splice(3, 1);  # OUTPUT: «[8]␤»
+say @a;               # OUTPUT: «[3 18 4 92 14 30]␤»
+=end code
+
+Note that the L<delete subscript|/language/subscripts#index-entry-:delete_(subscript_adverb)> replaces the element with its natural (i.e. default) state. Here (Any):
+
+=begin code
+my @a = <3 18 4 8 92 14 30>;
+@a[3]:delete;
+say @a;  # OUTPUT: «[3 18 4 (Any) 92 14 30]␤»
+=end code
+
+To delete by value, use L<.grep|/routine/grep>:
+
+=begin code
+my @letters = <a b c>;
+@letters .= grep: * ne "b";
+say @letters;  # OUTPUT: «[a c]␤»
+=end code
+
+=head2 List: How can I show index while looping through a list
+
+Use L<.kv|/routine/kv> or L<.pairs|/routine/pairs>:
+
+=begin code
+my @aa = 8 .. 12;
+for @aa.kv -> $i, $_ { say "$i: $_" };     # OUTPUT: «0: 8␤1: 9␤2: 10␤3: 11␤4: 12␤»
+say "$_.key(): $_.value()" for @aa.pairs;  # OUTPUT: «0: 8␤1: 9␤2: 10␤3: 11␤4: 12␤»
+=end code
+
+=head2 List: How can I concatenate lists
```

Comment:
I don't really think this is a FAQ. Come this point, I think you should really justify adding new questions to the FAQ. There's no issue in this particular area, these are simple operations that could be better explained or used as an example somewhere else.

Reply:
Removed

Agreed, like the head of pages before method declaration with some code examples : https://docs.raku.org/type/List#Immutability. I think they deserve more code example, wich is what people are looking for (I don't remember the post laughing on that in blog.perl).